083-stretch-cluster.md

```diff
@@ -0,0 +1,199 @@
+# Stretch Kafka cluster
+
+The Strimzi Kafka operator currently manages Kafka clusters within a single Kubernetes cluster.
+This proposal aims to extend support to stretch Kafka clusters, where brokers and controllers of a single Kafka cluster are distributed across multiple Kubernetes clusters.
+
+## Current situation
+
+At present, the availability of Strimzi-managed Kafka clusters is directly tied to the availability of the underlying Kubernetes cluster.
+If a Kubernetes cluster experiences an outage, the entire Kafka cluster becomes unavailable, disrupting all connected Kafka clients.
+
+## Motivation
+
+A stretch Kafka cluster allows Kafka nodes to be distributed across multiple Kubernetes clusters, significantly enhancing resilience by enabling the system to tolerate the outage of an entire Kubernetes cluster without disrupting service to clients.
+This configuration ensures high availability and seamless client operations, even in the event of cluster-specific failures.
+
+In addition to improving fault tolerance, this approach also facilitates other valuable use cases, such as:
+
+- **Migration Flexibility**: The ability to move Kafka clusters between Kubernetes environments without downtime, supporting maintenance or migrations.
```

Comment:
I would not consider just moving an entire Kafka cluster between Kubernetes envs but also only some nodes of the Kafka cluster itself, or?